components/world-4/breeding/pets.tsx

```diff
@@ -0,0 +1,522 @@
+                                <ShadowBox background="dark-1" key={pIndex} border={{ size: '2px', style: 'solid', color: 'green' }} direction="row" gap="medium" margin={{ bottom: 'medium', right: 'small' }} align="center" pad="small" style={{ opacity: pet.getBreedabilityBonus() >= 1.01 ? 1 : .5 }}>
+                                    <TipDisplay
+                                        size='medium'
+                                        maxWidth='medium'
+                                        heading={enemy?.data.details.Name ?? "Unknown"}
+                                        body={
+                                            <Box>
+                                                <Text size="medium">Currently in fenceyard : {petInFenceyard}</Text>
+                                                <Text size="medium">Total contribution per day : {nFormatter(petInFenceyard * breeding.getBreedingSpeed(starSignEquipped, silkRodeChip))}</Text>
+                                                {timeToNextLevel > 0 && <Text>Time to next level: {toTime(timeToNextLevel)}</Text>}
+                                            </Box>
+                                        }
+                                        direction={TipDirection.Down}
+                                    >
+                                        <IconImage data={{ location: enemy?.id.toLowerCase() ?? "Unknown", width: 67, height: 67 }} style={{ paddingBottom: '15px' }} />
+                                    </TipDisplay>
+                                    <Grid columns={["50%", "50%"]} fill align="center">
+                                        <Box>
+                                            <Text size="16px">Lvl: {pet.breedingLevel}</Text>
+                                            <Text size="16px">{nFormatter(pet.breedingProgress)}/{nFormatter(pet.getNextBreedingGoal())} days</Text>
+                                        </Box>
+                                        <Text size="16px">x{nFormatter(pet.getBreedabilityBonus())}</Text>
+                                    </Grid>
+                                </ShadowBox>
+                                :
+                                <ShadowBox background="dark-1" key={pIndex} direction="row" gap="medium" margin={{ bottom: 'medium', right: 'small' }} align="center" pad="small" style={{ opacity: pet.getBreedabilityBonus() >= 1.01 ? 1 : .5 }}>
+                                    <IconImage data={{ location: enemy?.id.toLowerCase() ?? "Unknown", width: 67, height: 67 }} style={{ paddingBottom: '15px' }} />
+                                    <Grid columns={["50%", "50%"]} fill align="center">
+                                        <Box>
+                                            <Text size="16px">Lvl: {pet.breedingLevel}</Text>
+                                            <Text size="16px">{nFormatter(pet.breedingProgress)}/{nFormatter(pet.getNextBreedingGoal())} days</Text>
```

Comment:
Same as the comment above for shiny, with the way it is now I have no idea how long I'll have to wait to get another level for the tome point.

Reply:
See replies to the above for shiny